statsmodels/robust/scale.py

```diff
@@ -81,6 +85,73 @@ def iqr(a, c=Gaussian.ppf(3/4) - Gaussian.ppf(1/4), axis=0):
         return np.squeeze(np.diff(quantiles, axis=0) / c)
 
 
+def qn_scale(a, c=1 / (np.sqrt(2) * Gaussian.ppf(5 / 8)), axis=0):
+    """
+    Computes the Qn robust estimator of scale, a more efficient alternative
```

Comment:
This needs to be split into a 1-line header, then a space, then an extended defintioin. For example,

```
Computes the Qn robust estimator of scale

The Qn scale estimator is a more efficient alternative to the MAD ...
```

Reply:
Done in b04b51cb2b83b3d9e1828d3b53956c1a7fd44629